server/code/setup/cpdb

```diff
@@ -1,4 +1,3 @@
-#!/usr/bin/python
```

Comment:
Does pathfix add a shebang or simply fix existing shebangs?

Reply:
You're right, it fixes existing shebangs, sorry about that.